test/index.html

```diff
@@ -100,7 +100,9 @@
         })(simpleName);
     }
 
-    jwplayer('video-container').setup(config);
+    var jwp = jwplayer('video-container');
+    jwp.setVolume('40');
+    jwp.setup(config);
 
```

Comment:
if you play, then setup again, what will happen without api.remove() we probably want a brand new internal controller and model, but may be able to reuse api and controller if they are stateless (watch out for that queue).